### PR TITLE
[Files] Add landing page tests

### LINKED ISSUE #2114: Add new ui test coverage for landing page in Files
As described, add test coverage for the below scenarios:

- Navigate to the Privacy policy page
- Navigate to the Terms & Conditions page
- Navigate to ChainSafe.io from "Learn more about Chainsafe"
- Accept cookie banner on main page

### PR DESCRIPTION
**Pull request by juans-chainsafe**
closes #2114

### CODE REVIEW

File: packages/files-ui/src/Components/Modules/LoginModule/InitialScreen.tsx [hunk 1]
```diff
@@ -537,6 +537,7 @@ const InitialScreen = ({ className }: IInitialScreen) => {
                 href={ROUTE_LINKS.PrivacyPolicy}
                 target="_blank"
                 rel="noopener noreferrer"
+                data-cy="button-privacy-policy"
```
Comment (asnaith) [inline packages/files-ui/src/Components/Modules/LoginModule/InitialScreen.tsx]:
```suggestion
                data-cy="link-privacy-policy"
```

File: packages/files-ui/src/Components/Modules/LoginModule/InitialScreen.tsx [hunk 2]
```diff
@@ -546,6 +547,7 @@ const InitialScreen = ({ className }: IInitialScreen) => {
                 href={ROUTE_LINKS.Terms}
                 target="_blank"
                 rel="noopener noreferrer"
+                data-cy="button-terms-and-conditions"
```
Comment (asnaith) [inline packages/files-ui/src/Components/Modules/LoginModule/InitialScreen.tsx]:
```suggestion
                data-cy="link-terms-and-conditions"
```

File: packages/files-ui/src/Components/Pages/LoginPage.tsx [hunk 3]
```diff
@@ -182,6 +182,7 @@ const LoginPage = () => {
         href={ROUTE_LINKS.ChainSafe}
         target="_blank"
         rel="noopener noreferrer"
+        data-cy="button-learn-more-about-chainsafe"
```
Comment (asnaith) [inline packages/files-ui/src/Components/Pages/LoginPage.tsx]:
```suggestion
        data-cy="link-learn-more-about-chainsafe"
```

File: packages/files-ui/cypress/support/page-objects/authenticationPage.ts [hunk 4]
```diff
@@ -13,6 +13,9 @@ export const authenticationPage = {
   showMoreButton: () => cy.get("div.svelte-q1527 > .bn-onboard-custom"),
   detectedWallet: () => cy.get(":nth-child(3) > .bn-onboard-custom > span.svelte-1799bj2"),
   web3SignInButton: () => cy.get("[data-cy=button-sign-in-with-web3]"),
+  privacyPolicyButton: () => cy.get("[data-cy=button-privacy-policy]"),
```
Comment (asnaith) [inline packages/files-ui/cypress/support/page-objects/authenticationPage.ts]:
```suggestion
  privacyPolicyLink: () => cy.get("[data-cy=link-privacy-policy]"),
```

File: packages/files-ui/cypress/support/page-objects/authenticationPage.ts [hunk 5]
```diff
@@ -13,6 +13,9 @@ export const authenticationPage = {
...
+  termsAndConditionsButton: () => cy.get("[data-cy=button-terms-and-conditions]"),
```
Comment (asnaith) [inline packages/files-ui/cypress/support/page-objects/authenticationPage.ts]:
```suggestion
  termsAndConditionsLink: () => cy.get("[data-cy=link-terms-and-conditions]"),
```

File: packages/files-ui/cypress/tests/landing-spec.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+import { localHost } from "../fixtures/loginData"
+
+describe("Landing", () => {
+  beforeEach(() => {
+    cy.visit(localHost)
+  })
+  context("desktop", () => {
+
+    it("can navigate to privacy policy page", () => {
+      authenticationPage.privacyPolicyButton().invoke('removeAttr', 'target').click()
```
Comment (asnaith) [inline packages/files-ui/cypress/tests/landing-spec.ts]:
```suggestion
      authenticationPage.privacyPolicyLink().invoke('removeAttr', 'target').click()
```

File: packages/files-ui/cypress/tests/landing-spec.ts [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+      cy.url().should("include", "/privacy-policy")
+    })
+
+    it("can navigate to terms & conditions page", () => {
+      authenticationPage.termsAndConditionsButton().invoke('removeAttr', 'target').click()
```
Comment (asnaith) [inline packages/files-ui/cypress/tests/landing-spec.ts]:
```suggestion
      authenticationPage.termsAndConditionsLink().invoke('removeAttr', 'target').click()
```

File: packages/files-ui/cypress/tests/landing-spec.ts [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+      cy.url().should("include", "/terms-of-service")
+    })
+
+    it("can navigate to ChainSafe.io from 'Learn more about Chainsafe'", () => {
+      authenticationPage.learnMoreAboutChainsafeButton().invoke('removeAttr', 'target').click()
```
Comment (asnaith) [inline packages/files-ui/cypress/tests/landing-spec.ts]:
```suggestion
      authenticationPage.learnMoreAboutChainsafeLink().invoke('removeAttr', 'target').click()
```

**Review by asnaith — APPROVED** [review]
Awesome, 2nd week and already adding more test coverage 🙌

These look good, now these links can't break without us knowing about it :)

**Review by FSM1 — APPROVED** [review]

**Review by tanmoyAtb — APPROVED** [review]
💯 💯

**Pull request merged**